source/tomopy/prep/stripe.py

```diff
@@ -939,3 +941,156 @@ def _remove_stripe_based_interpolation(tomo, snr, size, drop_ratio, norm):
         sino = tomo[:, m, :]
         sino = _rs_interpolation(sino, snr, size, drop_ratio, norm)
         tomo[:, m, :] = sino
+
+
+def stripes_detect3d(arr, vert_filter_size_perc=5, radius_size=3, ncore=None):
```

Comment:
Please change the `arr` argument so it is consistent with the other stripe methods. i.e. It should be named `tomo` and have the same axis ordering as the other methods in this module (I'm actually not sure what that is because the docs aren't great).

Reply:
fixed in 4974eee 
I renamed input `arr` to `tomo` for one function and to `weights` for another. Also I checked the conventional axis ordering for `tomo` and it seems to be consistent with` [angles, vertical_detector, horizontal_detector]` .